### ISSUE TITLE
[Feature request] list of domains in subject alt names

### ISSUE THREAD

**Issue by Albirew** [jannispinter/indicatetls#16]
Is it possible to list all (sub)domains in subject alt names?
(*.domain.tld for wildcard cert or a list with www.domain1.tld, mail.domain1.tld, www.domain2.tld, etc. if all using the same cert)
I believe it's possible since "Certainly Something" does it.

**Comment by jannispinter** [comment]
Yes, but that requires me to parse the certificate with [ASN1.js ](https://asn1js.org/) and [PKI.js](https://pkijs.org/) (as Certainly Something does). 

For now, I have not included any third party code and the Firefox API won't list the individual SAN entries. But this is definitively planed and I too would like to see the SANs.

**Comment by jannispinter** [comment]
Parsing the certificate is also required to get the key type and key length which is currently not shown. But I will implement that soon.

**Comment by Albirew** [comment]
i'm currently cleaning my old issues that are still open.
since last activity is from 2020, should i close this issue?
